juju/juju
# Add labels to metrics.

## Description of change

> Why is this change needed?

Labels will enable aggregating KPI metrics by application-specific
criteria.

## QA steps

> How do we verify that the change works?

Deploy a charm that uses the new `add-metric --labels` option in its collect-metrics hook. Verify that metrics are collected with labels.

## Documentation changes

> Does it affect current user workflow? CLI? API?

Yes, the `add-metric --labels` option will be added to the Juju metrics documentation.

## Bug reference

> Does this change fix a bug? Please add a link to it.

N/A
